### ISSUE TITLE
"Recent Apps"-button does not show recent apps

### ISSUE THREAD

**Issue by homburg** [brave/browser-android#726]
Brave 1.9.14

The standard recent apps button only hides the brave browser, but does not show the recent apps list

https://fat.gfycat.com/WeeInfiniteGoral.webm

**Comment by SergeyZhukovsky** [comment]
@homburg Did you have any recent apps in your video? For example if you don't have any open apps - it shows nothing. But open Phone Settings for example, and after that, switch to brave and click that button. It should show you the Phone Settings.

**Comment by homburg** [comment]
Yes, I had lots of recent apps and chrome tabs. I should have demonstrated
that in the video.

Have you tried reproducing the error?

**Comment by SergeyZhukovsky** [comment]
Yep. It works fine for me.

**Comment by homburg** [comment]
Try it with the home screen/launcher behind brave (Google Launcher).

tir. 5. apr. 2016 kl. 12.08 skrev Serg <notifications@github.com>:

**Comment by homburg** [comment]
Seems to be specific to the home screen

https://fat.gfycat.com/GrimDenseJavalina.webm

**Comment by SergeyZhukovsky** [comment]
Do you have that behaviour only with your home screen? I've checked it again on home screen also with Google Launcher and it always shows me recent apps. Maybe it is specific to your device and Android version? I tried on 6.0.1 and 5.0. What device and Android version do you have?

**Comment by homburg** [comment]
The recent apps button works as expected if brave is in front of a regular app.

It is only an issue, when on the home screen.
https://fat.gfycat.com/GrimDenseJavalina.webm

Nexus 5, Android 6.0.1, Google Now Launcher 1.3.large
